src/NuGetSdkResolver/Resources/AssemblyResources.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Resources;
+using System.Reflection;
+using System.Globalization;
+
+namespace Microsoft.Build.Shared
+{
+    /// <summary>
+    /// This class provides access to the assembly's resources.
+    /// </summary>
+    internal static class AssemblyResources
```

Comment:
Can this be simplified for this assembly?

Reply:
Done